contracts/Challenge/Challenge.sol

```diff
@@ -5,8 +5,8 @@ interface Challenge {
   function passed() view returns (bool);
 
   // amount of tokens the Challenge will need to carry out operation
-  function requestedTokenAmount() view returns (uint256);
+  function requestedTokenAmount() view returns (uint);
 
   // amount of tokens the Challenge will return to registry after conclusion (ie: winner reward)
-  function returnTokenAmount() view returns (uint256);
+  function returnTokenAmount() view returns (uint);
```

Comment:
I've heard conflicting opinions on this .. but since `uint256` and `uint` are the same, some say it's better to use the explicit naming of `uint256`. @cwhinfrey probably has an opinion from looking at various audits that may have addresses this

Reply:
I know zeppelin called this out once but I’m all for using uint. It’s very rare that you get any benefit from using something smaller than uint256 and uint just reads cleaner to me. It’s my personal opinion though.

Reply:
I share this opinion!! (on cleaner to read). I however need to up my skills on data types & their pros/cons